src/componets/Orderform.jsx

```diff
@@ -0,0 +1,59 @@
+
+function Orderform({fruitCounts}) {
```

Comment:
Het formulier is niet echt herbruikbaar, Dus is het niet echt een component, je zou wel de inputvelden herbruikbaar kunnen maken (en als component)